packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/edit-display-name/template.html

```diff
@@ -0,0 +1,31 @@
+<form
+    name="updateNodeDisplayNameForm"
+    novalidate
+    data-ng-submit="$ctrl.onUpdateDisplayNameFormSubmit()"
+>
+    <oui-modal
+        data-primary-label="{{($ctrl.actionEnabled ? 'nutanix_node_server_displaname_change' : 'nutanix_node_server_displaname_Ok') | translate }}"
+        data-secondary-label="{{$ctrl.actionEnabled ? ('nutanix_node_server_displaname_cancel' | translate) : '' }}"
+        data-secondary-action="$ctrl.goBack()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.loading.update"
+        data-type="{{ !$ctrl.actionEnabled ? 'warning' : '' }}"
+        data-heading="{{ :: 'nutanix_node_server_displaname_edit_title' | translate }}"
```

Comment:
Same as this comment:
https://github.com/ovh/manager/pull/6306/files#r784630957

Reply:
Yes its normal. I am using form submit instead of primary action.